games.zetsubou.WanderNoMore.yml

```diff
@@ -0,0 +1,35 @@
+app-id: games.zetsubou.WanderNoMore
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: WanderNoMore.sh
+tags:
+  - proprietary
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --socket=pulseaudio
+  - --persist=.renpy
+modules:
+  - name: hmp
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/bin/
+      - cp -R * /app/bin/
```

Comment:
you should remove the file for the other architeture.

Reply:
Will do. Is the preferred method to use the "only-arches" attribute on the sources, so there's a different archive downloaded for aarch vs x86_64? Or is a different method of separating them preferred, such as a different module per architecture?

It appears there's a problem with the aarch64 build anyway. I've contacted the game engine's developers to hopefully get this fixed. I don't think it'll take long to fix, but it might be a few days before I can proceed.

Reply:
Using `only-arches` on specific sources is the best if build commands are the same, otherwise module-level only-arches is fine too.

Reply:
here the binary dist contain both. For once "64-bits" doesn't mean x86_64 only :-)

Use $FLATPAK_ARCH to determine which build is happening and just copy that. Might need a bit more line to copy all the content.

Reply:
For the time being I've used only-arches on the source, so it'll download either the aarch64 or x86_64 build depending on the target architecture.
I've tried building and running the flatpak on both x86_64 and aarch64 Ubuntu machines. The resulting flatpaks work on their respective machines, so I believe it should be okay.